README.md

```diff
@@ -3,134 +3,320 @@
+
+### Methods
 
-* Lets create a functino outside of the `Planet` class declaration, it's a global function in Playground
+You could rewrite the tuple version of `planetInfo` pretty easily to take in a `Planet` parameter instead, and print the information encapsulated by that `Planet` instance:
 
 ```swift
-func lostOxygen(self: Planet) {
-    
-    self.percentOxygen -= 0.02
-    
+func planetInfoObject(planet: Planet) {
+    var moonNoun = "moon"
+    if planet.numberOfMoons != 1 {
+        moonNoun = "moons"
+    }
+    var hasLifeString = "has life"
+    if !planet.hasLife {
+        hasLifeString = "does not have life"
+    }
+    print("\(planet.name) has \(planet.numberOfMoons) \(moonNoun). Its orbit is \(planet.orbitalPeriod) days. It \(hasLifeString).")
 }
 
-lostOxygen(earth)
+planetInfoObject(earth2)
+planetInfoObject(mars2)
 ```
 
-* Here we've named the parameter `self` which is of type `Planet`. So whomever calls on this function, must give us a `Planet`  and we refer to the `Planet` given to us within the scope of this function by our parameter named `self`. 
-* Within the scope of every `init` and `func` that you create within your class declaration, you have access to an implicit variable called `self` which refers to the instance of the object who is calling on those functions.
+Keep in mind, though, that `planetInfoObject` is defined globally, meaning it is available everywhere in your Swift program, and more important, that the name cannot be used by any other function. This isn't a problem for `Planet` necessarily, but what if you start accumulating more classes, each with its own set of functions. At some point, it would be nice to have more organization in your functions, wouldn't it?
+
+Luckily, Swift provides a way to do that with methods. A _method_ is a function that is tied to a particular class. You can call that method on instances of that class. Methods are declared just like functions, but within the scope (curly braces) of a class definition.
+
+Most importantly, a method automatically has access to the _instance_ that it was called on. That instance is referred to as `self` within that method body. You could move `planetInfoObject` to your `Planet` class definition like so:
 
 ```swift
 class Planet {
-    let name: String
-    let numberOfMoons: Int
-    var percentOxygen: Double
+    var name: String
+    var numberOfMoons: Int
+    var orbitalPeriod: Double
     var hasLife: Bool
-    
-    init(name: String, numberOfMoons: Int, percentOxygen: Double, hasLife: Bool) {
+
+    init(name: String, numberOfMoons: Int, orbitalPeriod: Double, hasLife: Bool) {
         self.name = name
         self.numberOfMoons = numberOfMoons
-        self.percentOxygen = percentOxygen
+        self.orbitalPeriod = orbitalPeriod
         self.hasLife = hasLife
     }
-    
-    func lostOxygen() {
-        self.percentOxygen -= 0.02
+
+    func planetInfo() {
```

Comment:
Them being able to reference the function above and the use of `self` is here is a really great idea.